tools/renew-certificates.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+
+gateway_dir="/home/pi/mozilla-iot/gateway"
+
+certbot renew \
+    --config-dir "$gateway_dir/etc" \
```

Comment:
Is there configuration that we need to specify?

Reply:
certbot will try to auto-renew any certificates it finds in the given config directory. When we generate certs, those are now written out to the filesystem, including the renewal config. See #363.